code/API_definitions/population-density-data.yaml

```diff
@@ -88,6 +88,23 @@ info:
     The API provides one endpoint that accepts POST requests for retrieving population
     density information in the specified area.
 
+    # Authorization and Authentication
+    CAMARA guidelines defines a set of authorization flows which can grant API clients access

```

Comment:
There should be an additional line break between the heading and the content; otherwise, the content does not display properly in the Swagger editor.

Reply:
Apart from this, I would suggest to have examples for all three options (SUPPORTED_AREA, PART_OF_AREA_NOT_SUPPORTED, and AREA_NOT_SUPPORTED) in the response (200) example. Currently, only an example for "SUPPORTED_AREA" is provided.

Reply:
both included now in #32 , can you please confirm there to merge in this same PR?